worlds/kh2/docs/en_Kingdom Hearts 2.md

```diff
@@ -49,13 +55,12 @@ For example, if you are fighting Roxas and you receive Reflect Element and you d
     1. Obtain Three Proofs.
     2. Obtain a desired amount of Lucky Emblems.
     3. Obtain a desired amount of Bounties that are on late locations.
-- Customize how many World Locking Items You Need to Progress in that World.
-- Customize the Amount of World Locking Items You Start With.
-- Customize how many locations you want on Sora's Levels.
-- Customize the EXP Multiplier of everything that affects Sora.
-- Customize the Available Abilities on Keyblades.
-- Customize the level of Progressive Movement (Growth Abilities) you start with.
-- Customize the amount of Progressive Movement (Growth Abilities) you start with.
+- Customize how many World-Locking Items you need to progress in that world.
+- Customize the amount of World-Locking Items you start with.
+- Customize how many of Sora's Levels are locations.
+- Customize the EXP multiplier for Sora.
```

Comment:
```suggestion
- Customize the EXP multiplier for Sora and his Drive Forms.
```
This still leaves out summon levels, but not sure there's a great way to word that.

Reply:
Most suggestions were committed.
I tried adding mention of summons in commit ce0ea5d25bf6b53238f4cadc47e000e45bb0b6f4 , see if this is what you are looking for @Silvris .